docs/ABOUT.md

```diff
@@ -1,41 +1,18 @@
-
-Haskell is a purely functional, lazy, statically-typed programming
-language with type inference.
+Haskell is a purely functional, lazy, statically-typed programming language with type inference. 
+It has a beautiful and minmal syntax.
```

Comment:
Note for reviewers: This sentence was added.

Reply:
I've added that as it was part of the sentence next to the code section (which has been removed as part of the new formatting guidelines). Feedback is welcome on the changes 😄

Reply:
Previously, *immediately* after the list of traits describing Haskell (purely functional, lazy, etc.), there comes a description of each trait. 

Now, there is this sentence in between. Does this interposition interfere with the reading and understanding of this text?

Alternative: Leaving this sentence about beautiful and minimal syntax where it originally was. Is that better or worse, without having a code example?

(I have not decided what my own answer to these two questions is yet, so I'll leave them for others to think about for now)